bank-sdk/example-app/src/main/java/net/gini/android/bank/sdk/exampleapp/ui/CaptureFlowHostActivity.kt

```diff
@@ -37,10 +45,18 @@ class CaptureFlowHostActivity : AppCompatActivity() {
             .startCaptureSDKForIntent(openWithIntent)
     }
 
+    private fun startBankSDKForDocument(document: Document) {
+        findViewById<FragmentContainerView>(R.id.fragment_host).getFragment<ClientBankSDKFragment>()
+            .startBankSDKForDocument(document)
+    }
+
     companion object {
         fun newIntent(context: Context, openWithIntent: Intent? = null) =
```

Comment:
[Suggestion] What do you think of creating a second `newIntent()` method which can be used for the document:
```
fun newIntent(context: Context, openWithDocument: Document)
```

It looks like that have three distinct entry points for this activity:
1. launch without extras
2. launch with "open with" Intent
3. launch with "open with" Document

We should then reflect this in the `CaptureFlowHostActivity`'s public API, no?